api/src/routes/team.js

```diff
@@ -12,4 +12,6 @@ teamRouter.patch('/:team_id/twitter-handle', teamMiddleware.validateTeamId, team
 
 teamRouter.get('/:team_id', teamMiddleware.validateTeamId, teamController.getTeam);
 
+teamRouter.patch('/:id',teamController.updateTeam);
+
```

Comment:
I see a controller for updating a team, but do we have one for creating a team?

Reply:
@ssha0054 I think we don't right? I think when they register we create a new User, not really a team.